unity-client/Assets/Scripts/MainScripts/DCL/Controllers/WorldBlockersController.cs

```diff
@@ -0,0 +1,46 @@
+﻿using UnityEngine;
+
+namespace DCL.Controllers
+{
+    public class WorldBlockersController : MonoBehaviour
```

Comment:
just out of curiosity, do we need this to be a `MonoBehaviour`? can't we implemented so it doesn't?

Reply:
We need a gameobject to be the parent of the blockers in-world and be repositioned when the world repositions... we could avoid having this class as a Monobehaviuor but that would require to move some logic into an existent one, like the SceneController, but we are refactoring it to take away all these kind of responsibilities right? So I'm not sure where else could we put the logic that deals with the parent of the blockers

Reply:
Ok, didn't see the reposition there... makes sense then 👍

Reply:
If that is the only reason, It isn't better to just make a `new GameObject` on the constructor and use that as the movable parent? This way we can mock tests by just doing `new WorldBlockersController()` and save the Unity specific overhead.

Reply:
OK I'll do that and I'll add the WorldBlockersController initialization in the SceneControlle then.

Reply:
Done, now the WorldBlockersConttoller is created with dependency injection (and the parent GO created on initialization).